packages/client/src/context/Auth.context.tsx

```diff
@@ -106,28 +116,59 @@ export const AuthProvider: FC<AuthProviderProps> = ({ children }) => {
     handleUnauthenticated();
   };
 
+  // Handle switch tab
+  const [activeTab, setActiveTab] = useState<string>('login');
+  const handleTabClick = (tab:string) => {
+    setActiveTab(tab);
+  };
+
+  const location = useLocation();
+  const isResetPassword = location.pathname === "/reset-password";   // determine if it is reset password page
+
   return (
-    <AuthContext.Provider value={{ token, authenticated, decodedToken, logout }}>
-      {!authenticated && organization && (
-        <FirebaseLoginWrapper setToken={handleAuthenticated} organization={organization} />
-      )}
-      {authenticated && children}
-    </AuthContext.Provider>
+    <div style={styles.container}>
```

Comment:
You should leverage MUI for all the styling, HTML element structure, and should not have any div containers.

Instead using MUI components like `Box`, `Container`, `Stack`, etc. That way they styling and formatting is consistent with the rest of the website.